src/shrutiC/assignment2/FacebookLogin.java

```diff
@@ -0,0 +1,33 @@
+package shrutiC.assignment2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class FacebookLogin {
+
+	public static void main(String[] args) throws InterruptedException {
```

Comment:
you should create separate function for login and pass the username & password to that method.